test-compile.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from compile import generate
+
+
+def generate_md():
```

Comment:
@uchitsa maybe it's better to create a new directory `fixtures/` and place pairs of files there:

```
fixtures/
  simple/
    input.md
    input.yml
    expected.md
  more-complex/
    input.md
    input.yml
    expected.md
```

Then, the test will just load the `.yml`, generate `.md` and compare it with `expected.md`. If they don't match, the test will fail